Chinese Academy of Sciences CNIC CSTR.js

```diff
@@ -0,0 +1,194 @@
+{
+	"translatorID": "c070e5a2-4bfd-44bb-9b3c-4be20c50d0d9",
+	"label": "Chinese Academy of Sciences CNIC CSTR",
+	"creator": "cheney",
+	"target": "^https?://www\\.cstr\\.cn",
+	"minVersion": "3.0",
+	"maxVersion": "",
+	"priority": 97,
+	"inRepository": true,
+	"translatorType": 12,
+	"browserSupport": "gcsibv",
+	"lastUpdated": "2022-05-11 08:00:39"
+}
+
+function detectSearch(item) {
+	return !!item.CSTR;
+}
+
+function doSearch(item) {
+	
+	let url;
+	if (item.CSTR) {
+		url = "https://www.cstr.cn/openapi/v2/pid-cstr-service/detail?identifier=" + item.CSTR;
+	}
+	else if (item.query) {

```

Comment:
This branch will never be run, since `item.CSTR` will always be truthy if `detectSearch` matches.

Reply:
Do you mean that the Zotero search box input does not support CSTR? Cause this item.CSTR to be empty all the time?